_mappy-breakpoints.scss

```diff
@@ -80,7 +80,7 @@ $mappy-queries: () !default;
   }
   $_return: (
     type: $type, 
-    media-string: inspect($media-string), 
+    media-string: $media-string, 
```

Comment:
Interesting. Can you test the latest version out? I've kept inspect because I had to convert the list into a string. Let me know?